### PR TITLE
Add LMS launch banner for the teacher homepage

### PR DESCRIPTION
**Pull request by TurnerRiley**
Adds an LMS launch banner to the teacher homepage. I followed [this PR](https://github.com/code-dot-org/code-dot-org/pull/58308) to set up the banner and [this PR](https://github.com/code-dot-org/code-dot-org/pull/58460) to put it behind a DCDO flag (the same one as [the other LMS banners PR](https://github.com/code-dot-org/code-dot-org/pull/59035): `show-updated-lms-content`).

### Banner (in LTR)
banner 1

### Banner (in RTL)
banner 1 rtl

### Responsively shifts to have no image when the screen size gets smaller
responsive small

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1899)
Figma: [here](https://www.figma.com/design/6uM9F0bR7j9mOWO5LB8nUx/LMS-Marketing-Launch?node-id=2693-3775&t=GxUFO9ALkbURPqXm-0)

## Testing story
Local testing and ensuring that the banner only shows up when the DCDO flag is flipped to `true`.

## Follow-up work
Flip the DCDO flag on the 17th.

### CODE REVIEW

File: dashboard/config/marketing/announcements.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "pages": {
-    "/home": "music-lab-launch",
+    "/home": "lms-launch",
```
Comment (kelbyhawn) [inline dashboard/config/marketing/announcements.json:3]: Changing this now will remove the Music Lab banner, and no banner will show up until the `show-updated-lms-content` DCDO flag is flipped. I assume we want to keep the Music Lab banner up until this LMS banner goes live, but @dmcavoy can you confirm?

File: shared/images/banners/banner-lms-launch.png
Comment (kelbyhawn) [inline shared/images/banners/banner-lms-launch.png:1]: Left a [comment](https://www.figma.com/design/6uM9F0bR7j9mOWO5LB8nUx?node-id=2613-966#811578592) in Figma!

File: dashboard/config/marketing/announcements.json (hunk 1 above)
Comment (dmcavoy) [reply dashboard/config/marketing/announcements.json:3]: We've had the music lab banner up for 2 weeks. I think its alright for it to come down when this goes in

Comment (kelbyhawn) [reply dashboard/config/marketing/announcements.json:3]: Ok cool, ty! @TurnerRiley this is fine as-is 👍

File: shared/images/banners/banner-lms-launch.png
Comment (TurnerRiley) [reply shared/images/banners/banner-lms-launch.png:1]: Done and updated the PR pics!

**Review by kelbyhawn — APPROVED** [review]
🎉

**Pull request merged**